src/sql/base/browser/ui/table/plugins/buttonColumn.plugin.ts

```diff
@@ -39,7 +39,7 @@ export class ButtonColumn<T extends Slick.SlickData> extends BaseClickableColumn
 				return `<button tabindex=-1 class="${iconCssClasses} ${buttonTypeCssClass}" title="${escapedTitle}" aria-label="${escapedTitle}">${buttonText}</button>`;
 			},
 			name: this.options.name,
-			resizable: this.options.showText === true, // Image only button has fixed width.
+			resizable: this.options.resizable ?? this.options.showText === true,
```

Comment:
`resizable: this.options.resizable ` is good enough, no need to involve showText here.